docs/config.c

```diff
@@ -3717,7 +3718,7 @@
 ** .pp
 ** If \fIset\fP, a shell command to be executed if NeoMutt fails to establish
 ** a connection to the server. This is useful for setting up secure
-** connections, e.g. with \fCssh(1)\fP. If the command returns a  nonzero
+** connections, e.g. with \fIssh(1)\fP. If the command returns a  nonzero
```

Comment:
Hmmm, I haven't checked the others.  The `(1)` part should be roman.  The \fI\fP should enclose only the name of the page, and not the parenthesized number.  Please check all such occurences.